src/components/pill-selector-dropdown/PillSelectorDropdown.js

```diff
@@ -36,6 +36,8 @@ type Props = {
     onBlur: (event: SyntheticInputEvent<HTMLInputElement>) => void,
     /** Should update selectorOptions based on the given input value */
     onInput: Function,
+    /** Called when creating pill selector */
```

Comment:
Is the description accurate? creating pills instead?

Reply:
make sense